string action
string fulfillment_text
string parameters